src/typeguard/_decorators.py

```diff
@@ -31,6 +27,60 @@ def typeguard_ignore(f: _F) -> _F:
 T_CallableOrType = TypeVar("T_CallableOrType", bound=Callable[..., Any])
 
 
+def make_cell():
+    value = None
+    return (lambda: value).__closure__[0]
+
+
+def instrument(f: T_CallableOrType) -> Callable | str:
+    if not getattr(f, "__annotations__", None):
```

Comment:
Why not use `hasattr`?

Reply:
I suppose this also catches the case where `__annotations__ is None`?

Reply:
> I suppose this also catches the case where `__annotations__ is None`?

Indeed.